src/schemas.py

```
@@ -216,6 +216,14 @@ class DocumentUpdate(DocumentBase):
     metadata: dict[str, Any] | None = None
 
 
+class MessageSearchOptions(BaseModel):
+    query: str = Field(..., description="Search query")
+    use_semantic_search: bool = Field(
+        False,
```

Ruff (0.11.9)

Boolean positional value in function call

(FBT003)